jina/resources/k8s/template/configmap.yml

```diff
@@ -0,0 +1,8 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: config-map
+data:
+  jina_log_level: "DEBUG"
```

Comment:
this comes in capital letters no?

Reply:
i remembered is a common practice to use lowercase in configmap. It will be converted in upper case automatically.

Reply:
okk